<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPECharacteristics</class>
 <widget class="QDialog" name="DialogPECharacteristics">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>337</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>427</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>427</width>
    <height>337</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Characteristics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBoxX" name="checkBoxReadOnly">
       <property name="text">
        <string>Read only</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEditX" name="lineEditCharacteristics">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QCheckBoxX" name="checkBoxDEBUG_STRIPPED">
        <property name="text">
         <string notr="true">DEBUG_STRIPPED</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBoxX" name="checkBoxBYTES_REVERSED_HI">
        <property name="text">
         <string notr="true">BYTES_REVERSED_HI</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBoxX" name="checkBoxSYSTEM">
        <property name="text">
         <string notr="true">SYSTEM</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBoxX" name="checkBoxRELOCS_STRIPPED">
        <property name="text">
         <string notr="true">RELOCS_STRIPPED</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBoxX" name="checkBox32BIT_MACHINE">
        <property name="text">
         <string notr="true">32BIT_MACHINE</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBoxX" name="checkBoxAGGRESIVE_WS_TRIM">
        <property name="text">
         <string notr="true">AGGRESIVE_WS_TRIM</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBoxX" name="checkBoxDLL">
        <property name="text">
         <string notr="true">DLL</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBoxX" name="checkBoxREMOVABLE_RUN_FROM_SWAP">
        <property name="text">
         <string notr="true">REMOVABLE_RUN_FROM_SWAP</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBoxX" name="checkBoxEXECUTABLE_IMAGE">
        <property name="text">
         <string notr="true">EXECUTABLE_IMAGE</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBoxX" name="checkBoxLARGE_ADDRESS_AWARE">
        <property name="text">
         <string notr="true">LARGE_ADDRESS_AWARE</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBoxX" name="checkBoxBYTES_REVERSED_LO">
        <property name="text">
         <string notr="true">BYTES_REVERSED_LO</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBoxX" name="checkBoxNET_RUN_FROM_SWAP">
        <property name="text">
         <string notr="true">NET_RUN_FROM_SWAP</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBoxX" name="checkBoxUP_SYSTEM_ONLY">
        <property name="text">
         <string notr="true">UP_SYSTEM_ONLY</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBoxX" name="checkBoxLOCAL_SYMS_STRIPPED">
        <property name="text">
         <string notr="true">LOCAL_SYMS_STRIPPED</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBoxX" name="checkBoxLINE_NUMS_STRIPPED">
        <property name="text">
         <string notr="true">LINE_NUMS_STRIPPED</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonApply">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditX</class>
   <extends>QLineEdit</extends>
   <header>qlineeditx.h</header>
  </customwidget>
  <customwidget>
   <class>QCheckBoxX</class>
   <extends>QCheckBox</extends>
   <header>qcheckboxx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
